WORKSPACE

```diff
@@ -0,0 +1,33 @@
+load(
+    "//:repositories.bzl",
+    "boringssl_repositories",
+    "protobuf_repositories",
+    "googletest_repositories",
+    "libevent_repositories",
+    "spdlog_repositories",
+    "tclap_repositories",
+    "lightstep_repositories",
+    "http_parser_repositories",
+    "rapidjson_repositories",
+    "nghttp2_repositories",
+)
+
+boringssl_repositories()
+
+protobuf_repositories()
+
+googletest_repositories()
+
+libevent_repositories()
+
+spdlog_repositories()
+
+tclap_repositories()
+
+lightstep_repositories()
+
+http_parser_repositories()
+
+rapidjson_repositories()
+
+nghttp2_repositories()
```

Comment:
Could you add `envoy_dependencies()` (or `envoy_repositories()`) function that calls all of those functions to `repositories.bzl` to avoid polluting `WORKSPACE` file?